### ISSUE TITLE
Not building: no matching function for call to ‘entwine::Reader::query(...

### ISSUE THREAD

**Issue by tomvantilburg** [hobu/greyhound#33]
Running into a compilation error on:
- Ubuntu 16.04 64bit
- PDAL 1.3.0 with lazperf
- entwine build from master without errors in /usr/local/include/entwine/
- latest greyhound pull from master

```
> greyhound-server@0.0.7 install /usr/local/lib/node_modules/greyhound-server
> node-gyp rebuild

make: Entering directory '/usr/local/lib/node_modules/greyhound-server/build'
  CXX(target) Release/obj.target/session/src/session/bindings.o
  CXX(target) Release/obj.target/session/src/session/session.o
../src/session/session.cpp: In member function ‘std::shared_ptr<ReadQuery> Session::query(const entwine::Schema&, bool, double, const entwine::Point&, const entwine::Bounds*, std::size_t, std::size_t)’:
../src/session/session.cpp:203:31: error: no matching function for call to ‘entwine::Reader::query(const entwine::Schema&, const entwine::Bounds&, const size_t&, const size_t&, const double&, const entwine::Point&)’
                         offset)));

```

**Comment by connormanning** [comment]
Oops, the Greyhound build was out of sync with Entwine's API.  I've merged #32, which syncs them, so please re-pull and try again.  A fresh docker image is currently being pushed (very slowly), and the npm package has been updated.

Can you also clarify - what was your build command to get this error?

**Comment by tomvantilburg** [comment]
Yes, works!
Build was just the npm install on the latest git master, I've attached the full output.
[stacktrace_greyhound.txt](https://github.com/hobu/greyhound/files/508678/stacktrace_greyhound.txt)
